@xen-orchestra/fs/src/fs.spec.js

```diff
@@ -310,5 +310,17 @@ handlers.forEach(url => {
         await handler.unlink('file')
       })
     })
+
+    describe('#write()', () => {
+      beforeEach(() => handler.outputFile('file', TEST_DATA))
+
+      testWithFileDescriptor('file', 'r+', async ({ file }) => {
+        const WRITE_SIZE = 256
+        const WRITE_TEST_DATA = unsecureRandomBytes(WRITE_SIZE)
```

Comment:
Move this outside the test as it can be shared with both runs (with path and with file descriptor).